scripts/imgtool/main.py

```diff
@@ -254,6 +254,10 @@ def convert(self, value, param, ctx):
               type=click.Choice(['128','256']),
               help='When encrypting the image using AES, select a 128 bit or '
                    '256 bit key len.')
+@click.option('-c', '--clear', required=False, is_flag=True, default=False,
+              help='Output an encrypted image file that can be installed on '
+                   'primary slot. (clear image code with encrypt capability '
+                   'in secondary slot during swap.)')
```

Comment:
Technically speaking it's not an "encrypted image", I would suggest something like: "Output a non-encrypted image with encryption capabilities, so it can be installed in the primary slot, and encrypted when swapped to the secondary."